### DISCUSSION TITLE
Open-source contributions in Oct 2022

### DISCUSSION THREAD

**Discussion by darshitac11** [Atri-Labs/atrilabs-engine#393] [Community Shoutouts]
This project follows the [all-contributors](https://allcontributors.org/) specification. [Contributions of all types](https://allcontributors.org/docs/en/emoji-key) are welcome!

**Comment by darshitac11** [comment]
Thanks @harishfalco on becoming an open-source code contributor to the Atri framework today! 🎉 

In the upcoming release, we will have

- tooltips for the icons in the top panel of the visual editor
- keyboard shortcut for `Build and Run`

That’s all because of Harish who took the initiative to work on these dx improvements. ❤️

**Comment by darshitac11** [comment]
Congrats to @Ankittripa on becoming an open-source contributor to the Atri framework today! 🎉

He has made many valuable contributions such as:
- suggesting new features and ideas for DX improvement
- mentoring newcomers
- reporting bugs
- answering questions in Slack community

**Comment by Ankitsgithubs** [comment]
Thank you so much, ma'am!!!😄
